openhcl/underhill_core/src/nvme_manager.rs

```diff
@@ -407,6 +425,7 @@ impl NvmeManagerWorker {
                 vfio_device,
                 &disk.driver_state,
                 self.is_isolated,
+                None, // controller_instance_id is not persisted in saved state
```

Comment:
This is unfortunate. Is there a way to re-hook-up the `name` (formerly known as `controller_instance_id` and `debug_id`) after a restore, without putting it in the `NvmeManagerSavedState`?)